.github/workflows/ci.yaml

```diff
@@ -93,6 +93,9 @@ jobs:
   drenv-test:
     name: drenv tests
     runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
     steps:
       - name: Checkout source
         uses: actions/checkout@v3
```

Comment:
We need the set up the python-version option of the setup-python - see bellow this line.